chore: website fixes (#19)
IBM/ado: website/docs/resources/discovery-spaces.md

### Pre-requisites

In order to create a `discoveryspace` you must provide a `samplestore` that the `discoveryspace` will use for storage.

> To see existing `samplestores` run

Reviewer: ```suggestion
To see existing `samplestores` run:
```

```commandline
ado get samplestores
```

> Alternatively if there is no existing stire then when creating the space you can use the `---new-sample-store` flag.

Reviewer: ```suggestion
Alternatively, if there is no existing store, when creating the space you can use the `---new-sample-store` flag.
```

### Differences between input configuration YAML and stored configuration YAML

> After creating a `discoveryspace` if you `ado get` its YAML you will notice there is different information output than you input.

Reviewer: ```suggestion
After creating a `discoveryspace`, if you `ado get` its YAML you will notice that the information output is different from the one you provided in input.
```

If someone uses method one on `discoveryspace` A, they will only see the `entities` placed there by operations on `discoveryspace` A.
However if someone uses method two on `discoveryspace` A, they will see `entities` placed there via operations on both `discoveryspace` A and space B.

> Shared samples stores also allow data to be reused between `discoveryspaces` potentially accelerating exploration operations. 

Reviewer: ```suggestion
Shared samples stores also allow data to be reused across `discoveryspaces`, potentially accelerating exploration operations. 
```

This means the domains of the properties in the `entityspace` must be compatible with the experiments - if not entities could be sampled that
experiments in the `measurementspace` cannot measure. 

> For example to see the input requirements of the experiment `finetune_full_benchmark-v1.0.0` you can run

Reviewer: ```suggestion
For example, to see the input requirements of the experiment `finetune_full_benchmark-v1.0.0` you can run:
```

  finetune_full_benchmark-v1.0.0-gpu_compute_utilization_max
  finetune_full_benchmark-v1.0.0-gpu_memory_utilization_min
  finetune_full_benchmark-v1.0.0-gpu_memory_utilization_avg
  finetune_full_benchmark-v1.0.0-gpu_memory_utilization_max
  finetune_full_benchmark-v1.0.0-gpu_memory_utilization_peak
  finetune_full_benchmark-v1.0.0-gpu_power_watts_min
  finetune_full_benchmark-v1.0.0-gpu_power_watts_avg
  finetune_full_benchmark-v1.0.0-gpu_power_watts_max
  finetune_full_benchmark-v1.0.0-gpu_power_percent_min
  finetune_full_benchmark-v1.0.0-gpu_power_percent_avg
  finetune_full_benchmark-v1.0.0-gpu_power_percent_max
  finetune_full_benchmark-v1.0.0-cpu_compute_utilization
  finetune_full_benchmark-v1.0.0-cpu_memory_utilization
  finetune_full_benchmark-v1.0.0-train_runtime
  finetune_full_benchmark-v1.0.0-train_samples_per_second
  finetune_full_benchmark-v1.0.0-train_steps_per_second
  finetune_full_benchmark-v1.0.0-train_tokens_per_second
  finetune_full_benchmark-v1.0.0-train_tokens_per_gpu_per_second
  finetune_full_benchmark-v1.0.0-dataset_tokens_per_second
  finetune_full_benchmark-v1.0.0-dataset_tokens_per_second_per_gpu
  finetune_full_benchmark-v1.0.0-is_valid

```

> You can see the required inputs under the section `Required Inputs` and the optional inputs under `Optional Inputs and Default Values'.

Reviewer: ```suggestion
You can see the required inputs under the section `Required Inputs` and the optional inputs under `Optional Inputs and Default Values`.
```
